### PR TITLE
Created initial ui and activity for wear app [Story #91] [Task #92 #93]

### PR DESCRIPTION
**Pull request by almiche**
**Story:** #91 
**Tasks:** #92 #100 

### WHAT kind of change does this PR introduce?
Added a new user interface and activity for the Android Wear Platform. Users can input numbers into the watch calculator.

### Mockup
androidwearui

### HOW is this accomplished?

* A new `MainActivity` is defined for watch operations
* New XML files for the watch GUI
* All the calculator functionality is shared with the phone application via commons.

### Checklist
- [ ] Docs have been added / updated to reflect the changes
- [ ] I have reviewed and tested the changes :white_check_mark:
- [ ] I have squashed my commits to have a reasonable amount of commits
- [ ] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

File: wear/src/main/java/com/example/wear/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    override fun getResult(): String {
+        return result.text.toString()
+    }
+
+    override fun getFormula(): String {
+        return ""
+    }
+
+
```
Comment (izconcept) [inline wear/src/main/java/com/example/wear/MainActivity.kt]: **Style Criticism:** There should be 1 line between functions in Kotlin according to our coding convention.

File: wear/src/main/java/com/example/wear/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    private fun getButtonIds() = arrayOf(btn_decimal, btn_0, btn_1, btn_2, btn_3, btn_4, btn_5, btn_6, btn_7, btn_8, btn_9)
+
+    private fun checkHaptic(view: View) {
+        if (vibrateOnButtonPress) {
+            view.performHapticFeedback()
+        }
+    }
+
+
+    private fun updateButtonColor() {
```
Comment (izconcept) [inline wear/src/main/java/com/example/wear/MainActivity.kt]: This function seems obscure, what does it do?

File: wear/src/main/java/com/example/wear/MainActivity.kt (hunk 1 above)
Comment (almiche) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: Done

File: wear/src/main/java/com/example/wear/MainActivity.kt (hunk 2 above)
Comment (almiche) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: Legacy code that is no longer in use. Good catch Kevin!

Comment (izconcept) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: Ok, so I guess we'll remove it for now?

File: wear/src/main/java/com/example/wear/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    private fun getButtonIds() = arrayOf(btn_decimal, btn_0, btn_1, btn_2, btn_3, btn_4, btn_5, btn_6, btn_7, btn_8, btn_9)
+
+    private fun checkHaptic(view: View) {
+        if (vibrateOnButtonPress) {
+            view.performHapticFeedback()
+        }
+    }
+
+    private fun updateButtonColor() {
```
Comment (miisaka) [inline wear/src/main/java/com/example/wear/MainActivity.kt]: is this method used at all?

File: wear/src/main/java/com/example/wear/MainActivity.kt (hunk 1 above)
Comment (izconcept) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: Nice!

File: wear/src/main/res/values-round/strings.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="hello_world">Hello Round World!</string>
```
Comment (miisaka) [inline wear/src/main/res/values-round/strings.xml]: what is this for?

Comment (almiche) [reply wear/src/main/res/values-round/strings.xml]: This is automatically generated when adding a wear module.

Comment (miisaka) [reply wear/src/main/res/values-round/strings.xml]: Alright, we should clean up useless code

File: wear/src/main/java/com/example/wear/MainActivity.kt (hunk 3 above)
Comment (almiche) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: I've already removed this.

Comment (miisaka) [reply wear/src/main/java/com/example/wear/MainActivity.kt]: nvm XD you removed it

**Pull request merged**